main.less

```diff
@@ -30,16 +25,24 @@
     border: 1px solid #deca7e;
 }
 
-.errors {
-    width: 25px;
-    cursor: pointer;
-}
+.hint-marker-positioning {
+  left: auto !important; /* CodeMirror will set this value, but we don't want it set! */
+  right: 1em;
+  background-color: transparent;
 
-.gutterCursor {
-    cursor: pointer;
+  margin: 0px;
```

Comment:
I didn't write this css so I didn't want to touch it, but, if it isn't asking too much, is there a reason why?